picard/file.py

```diff
@@ -296,6 +299,16 @@ def _save_and_rename(self, old_filename, metadata):
             return None
         new_filename = old_filename
         if not config.setting["dont_write_tags"]:
+            # Copy file to memory
+            try:
+                num_bytes = os.stat(old_filename).st_size
+                source = open(old_filename, 'rb')
+                temporary_memory_file = source.read(num_bytes)
+            except FileNotFoundError as e:
+                log.debug("Failed to backup %s before saving the new tags.", old_filename)
```

Comment:
I'm aware Picard's code is not consistent in this, but
```suggestion
                log.debug("Failed to backup '%s' before saving the new tags.", old_filename)
```

Reply:
I've changed them to be more in line with the other log formats: "something happened with file : $filename".